src/components/activity/ActivityDirectivesTablePanel.svelte

```diff
@@ -64,6 +73,33 @@
       resizable: true,
       sortable: true,
     },
+    derived_start_time: {
+      field: 'derived_start_time',
+      filter: 'text',
+      headerName: 'Derived Start Time',
+      hide: true,
```

Comment:
Could we also default all of these new fields to showing? They are more intuitive than the offset.